### PR TITLE
Update general_tools.rst to mention Layer Definition Files.

### LINKED ISSUE #2111: [FEATURE] Show qlr files in browser
Original commit: https://github.com/qgis/QGIS/commit/fcf2fcae5abfb909288a65f2872746495b3a3fce by nyalldawson

Fixes #10737

### PR DESCRIPTION
**Pull request by havatv**
Fixes #2111.

- [x] Backport to LTR documentation is required

### CODE REVIEW

**Comment by DelazJ** [comment]
I'm a bit uncomfortable with the location of the text: added as an introduction to the "[save and share layer properties](https://docs.qgis.org/3.4/en/docs/user_manual/introduction/general_tools.html#save-and-share-layer-properties)", while it's not the main part of this section. IMO it should be added as a subsection after the existing ones (if we don't have much to tell about)
Also, with adding a new section header, it could be nice to add an anchor we would link to, from the action under [Layer menu](https://docs.qgis.org/3.4/en/docs/user_manual/introduction/qgis_gui.html#layer)

**Comment by havatv** [comment]
I tried with a separate section. I have also added references from the GUI description.

**Comment by SrNetoChan** [comment]
You can export layer and groups of layers into a Layer Definition File. Maybe we could mention that?

**Comment by havatv** [comment]
@SrNetoChan, I have tried update the text.  Please have a look.

File: source/docs/user_manual/introduction/general_tools.rst [hunk 1]
```diff
@@ -1694,6 +1694,21 @@ with name and description.
...
+.. index:: Layer Definition File, qlr file
+.. _layer_definition_file:
+
+Layer definition file
+--------------------- 
+
+Layer definitions can be saved as a ``Layer Definition File``
+(:file:`.qlr`) using
+:menuselection:`Export --> Save As Layer Definition File...` in the
+layer's context menu.
```
Comment (SrNetoChan) [inline source/docs/user_manual/introduction/general_tools.rst]:
```suggestion
active layers' context menu.
```

File: source/docs/user_manual/introduction/general_tools.rst [hunk 2]
```diff
@@ -1694,6 +1694,21 @@ with name and description.
...
+A layer definition file (:file:`.qlr`) includes references to the
+data source of the layers and their styles.
+:file:`.qlr` files are shown in the Browser Panel and can be used to add
```
Comment (SrNetoChan) [inline source/docs/user_manual/introduction/general_tools.rst]:
```suggestion
The :file:`.qlr` files are shown in the Browser Panel and can be used to add
```

File: source/docs/user_manual/introduction/general_tools.rst [hunk 3]
```diff
@@ -1694,6 +1694,21 @@ with name and description.
...
+the layers (with the saved style) to the Layers Panel.
+
```
Comment (SrNetoChan) [inline source/docs/user_manual/introduction/general_tools.rst]:
```suggestion
Alternatively, you can drag and drop :file:`.qlr` files from the system file manager into the map canvas to load.
```

File: source/docs/user_manual/introduction/general_tools.rst (hunk 2 above)
Comment (havatv) [reply source/docs/user_manual/introduction/general_tools.rst]: Thanks!  I modified / simplified a bit.

**Pull request merged**